cmd/flowrunner/testdata/flows/no_contact_test.json

```diff
@@ -33,7 +33,14 @@
                 },
                 {
                     "created_on": "2018-07-06T12:30:05.123456789Z",
-                    "fatal": true,
+                    "fatal": false,
```

Comment:
chatted with @ericnewcomer about this briefly and we felt an action that requires a contact being run in a session without a contact should just a log an error and move on